content/docs/user-guide/troubleshooting.md

```diff
@@ -91,3 +91,18 @@ using:
 ```dvc
 $ dvc checkout --relink
 ```
+
+## Invalid authentication for Git Remote {#git-auth}
```

Comment:
Is this an actual link output from `dvc` CLI?

Reply:
Ah yes I see it's in https://github.com/iterative/dvc/pull/6493/files#diff-bf8bef3f4902b5642f7280464c2f9c52574fb2f20b37c0c8fce28075c09e2b24.